pkg/logger/logger_test.go

```diff
@@ -0,0 +1,197 @@
+// Copyright 2025 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logger
+
+import (
+	"testing"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
```

Comment:
Use `assert` instead of `require` when the test still can run without problems. This way we get multiple assert errors in the log instead of 1 by 1 after running and fixing them
```suggestion
	"github.com/stretchr/testify/assert"
```

Reply:
I'll use assert instead of require here after, thanks for the review!